connect/api/v1/recent_activity/views.py

```diff
@@ -15,7 +15,23 @@ class RecentActivityAPIView(views.APIView):
 
     def post(self, request):
 
-        if request.data.get("flow_organization"):
+        action = request.data.get("action")
+        entity = request.data.get("entity")
+        entity_name = request.data.get("entity_name")
+        user = User.objects.get(email=request.data.get("user"))
+
+        if(request.data.get("intelligence_id")):
```

Comment:
Assign the return value of the get to a variable as done in other cases above to avoid repetition and shorten the code!